src/App.css

```diff
@@ -1,104 +1,872 @@
-body {
+/* Global styles */
+* {
+  box-sizing: border-box;
   margin: 0;
   padding: 0;
+}
+
+body {
```

Comment:
Great job @ONJoseph 🤝

- [ ] I can't seem to run the project smoothly. Kindly fix the error below and resubmit the project for review.
![errors](https://user-images.githubusercontent.com/63917725/215276491-aa06e57e-b96f-4d00-8889-c35d9c75e5c3.PNG)

Reply:
Fixed. Thanks a lot! @1GORDON